securedrop/journalist.py

```diff
@@ -65,6 +65,13 @@ def get_source(filesystem_id):
 @app.before_request
 def setup_g():
     """Store commonly used values in Flask's special g object"""
+    if 'expires' in session and datetime.utcnow() >= session['expires']:
+        session.clear()
+        flash('You have been logged out due to inactivity', 'error')
+
+    session['expires'] = datetime.utcnow() + \
+        timedelta(minutes=getattr(config, 'SESSION_EXPIRATION_MINUTES', 30))
```

Comment:
It's good that this has a default of 30 minutes, but otherwise where is this config option defined? (x-linking with #1966). Or are you just assuming that it isn't defined?

Reply:
Correct. I'm assuming it isn't defined, and I use `getattr` everywhere. Should someone really want to override it, it will pick it up. The unit/integration tests assume it's undefined and have checks to `del` the field in the `finally` block to not affect other tests.